zulipterminal/config/unicode_characters.py

```diff
@@ -0,0 +1 @@
+BLACK_RIGHT_POINTING_TRIANGLE = '▶'
```

Comment:
We may want to use names that are more descriptive so that it is easier for our end-users to depict what modifies which UI element. Also, you could include other Unicode characters that are present in the app.

See also [#zulip-terminal > Unicode characters](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/Unicode.20characters/near/862659).

Reply:
@preetmishra We could do that but I thought it would be better to stick to standard names from wikipedia?
The problem with names related to UI elements is that the same characters may be reused in a lot of places, for e.g. lineboxes.

Reply:
@kaustubh-nair There are quite a few cases not included in commit - I thought you were going to put this into a separate PR/issue?

I agree with @preetmishra that we probably want semantic names rather than direct ones, ie. here the equivalent of 'stream-topic-separator' rather than the currently-used version. That would mean we could change in one place and all the code+tests should update, and ZT run with different characters.

Reply:
There are a lot of unicode characters in lineboxes so it will be a little time-consuming, hence a new PR would be better.

Reply:
@neiljp How would you specify semantic names for lines in lineboxes that uses a different style in each corner?
Also the same lines are used as pane seperators, hence I thought a general name would be better.

Reply:
I intend to remove this commit, opened #717